frontend/src/forms/ParcelDetailForm.test.tsx

```diff
@@ -199,21 +214,7 @@ describe('ParcelDetailForm', () => {
     });
   });
 
-  it('ParcelDetailForm renders correctly', () => {
-    const history = createMemoryHistory();
-    const tree = renderer
-      .create(
-        <Provider store={store}>
-          <Router history={history}>
-            <ParcelDetailForm secret="test" agencyId={1} parcelDetail={mockDetails[0]} />
-          </Router>
-        </Provider>,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-
-  it('ParcelDetailForm renders view-only correctly', () => {
+  xit('ParcelDetailForm renders view-only correctly', () => {
```

Comment:
you're ignoring this test here but you added a non-ignored copy of this test above.